### PR TITLE
radare2 repo issue #7527

### PR DESCRIPTION
**Pull request by dmwit**
A test for the problem described in [issue 7527](https://github.com/radare/radare2/issues/7527).

### CODE REVIEW

File: t.tools/r2/seek_undo_redo [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+# GPL 3+ - Copyright (C) 2014  pancake
+for a in . .. ../.. ; do [ -e $a/tests.sh ] && . $a/tests.sh ; done
+
+NAME='seek undo/redo'
+BROKEN=
+FILE=-
+ARGS=
+CMDS='e scr.null=1
+wx 1
```
Comment (radare) [inline t.tools/r2/seek_undo_redo]: wx 1 is wrong. wx expects expairs. use `wx 01`

Comment (radare) [inline t.tools/r2/seek_undo_redo]: the funny thing is that if you run all those commands in a single line it works as expected

Comment (radare) [inline t.tools/r2/seek_undo_redo]: can you add another test for `CMDS=wx 1;s 1;s-;s+;pb 8` ?

File: t.tools/r2/seek_undo_redo [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+s 1
+s-
+s+
+e scr.null=0
```
Comment (radare) [inline t.tools/r2/seek_undo_redo]: none of those seek commands show anything to stdout, no need to use scr.null

**Comment by dmwit** [comment]
Thanks for the feedback. All fixed, I believe.

**Comment by radare** [comment]
looks good! thanks

**Pull request merged**